src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+
+    private int firstSide;
+    private int secondSide;
+    private int heightIsoscelesTrapezoid;
```

Comment:
Consider renaming 'heightIsoscelesTrapezoid' to 'height' for simplicity and consistency with other variable names 📏